CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributors Guide
+
+
+### dfmt follows the neptune versioning scheme 
+
+The following is taken verbatim from
+https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-contributor.rst
+
+
+## Submitting Pull Requests
+
+When making a PR to a Neptune-versioned library, the choice of which branch to
+submit a PR against is important. A branch should be chosen as follows:
+
+* If your change is a bug fix, the PR should be submitted against the oldest
+  supported minor branch.
+* If your change is a new feature, refactoring, or deprecation, the PR should be
+  submitted against the oldest supported major branch.
+* If your change is a breaking change to the API, the PR should be submitted
+  against the next unreleased major branch.
```

Comment:
Given that dfmt hasn't reached 1.0.0 and we don't have a 1.x.x branch, this will be hard for contributors to follow.

Reply:
as dfmt has not reached 1.0 the next major is the next minor